server/shadow/Mac/mac_shadow.c

```diff
@@ -660,7 +660,7 @@ macShadowSubsystem* mac_shadow_subsystem_new()
 	return subsystem;
 }
 
-int Mac_ShadowSubsystemEntry(RDP_SHADOW_ENTRY_POINTS* pEntryPoints)
+FREERDP_API int Mac_ShadowSubsystemEntry(RDP_SHADOW_ENTRY_POINTS* pEntryPoints)
```

Comment:
That's exactly why I have PR #3167 :)

But you are right, please keep it in your PR for now :)
If #3167 is merged first, you can remove them in this PR.
If this PR is merged first, I can remove them in my PR. 
Just make sure FreeRDP compiles well at anytime :)